packages/editor-ui/src/components/ExecutionsList.vue

```diff
@@ -363,23 +357,9 @@ export default mixins(externalHooks, genericHelpers, executionHelpers, restApi,
 				}
 				return returnData;
 			},
-			combinedExecutionsCount(): number {
-				return 0 + this.activeExecutions.length + this.finishedExecutionsCount;
-			},
 			numSelected(): number {
```

Comment:
when I click load more, this number does not get updated
<img width="1440" alt="Bildschirmfoto 2023-03-06 um 09 51 57" src="https://user-images.githubusercontent.com/4711238/223062274-377af236-a650-4241-964b-30735887b4c5.png">

when I click load more, second page loads selected, click middle item in second page, the rest of list gets deselected
<img width="1440" alt="Bildschirmfoto 2023-03-06 um 09 53 36" src="https://user-images.githubusercontent.com/4711238/223062462-3027ef55-d0e8-42b6-8072-8e9e489a64c9.png">